### ISSUE TITLE
When swiching through applications, with the ALT+Tab, NVDA is reporting before the Application name unknown?

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#5167]
**Reported by Brendon22 on 2015-06-20 09:44**
When swiching through applications, with the ALT+Tab, NVDA is reporting before the Application name unknown?

NVDA also is stating to me when I move back to the desktop after I have closed an application, NVDA states unknown, then I get the message "UI thread???

Can this please be fixed?

**Comment by bhavyashah** [comment]
I used to have this one laptop (without ICloud) where NVDA said 'unknown' for a lot of things, making NVDA effectively useless and non-functional as a screen reader. I no longer have that now, and thus cannot contribute anything substantial towards testing. Any which ways, what should we do with this ticket, bearing in mind that developers are unable to reproduce this issue, there exist no means to contact the original reporter, and the last activity on this ticket took place almost two years ago? @jcsteh @leonardder @ehollig

**Comment by ehollig** [comment]
Closing because of reasons stated in https://github.com/nvaccess/nvda/issues/5167#issuecomment-318837891.
If the reasoning is because of the iCloud Control Panel, and someone can replicate this and provide steps to reproduce, we can reopen this ticket. As stated by others though, worksforme
